src/js/breadcrumbs/breadcrumbs-view.js

```diff
@@ -0,0 +1,25 @@
+;(function (define) {
+    'use strict';
+    define(['backbone', 'underscore', 'text!./breadcrumbs.underscore'],
+        function (Backbone, _, breadcrumbsTemplate) {
+            var BreadcrumbsView = Backbone.View.extend({
+                initialize: function (options) {
+                    this.template = _.template(breadcrumbsTemplate);
+                    this.listenTo(this.model, 'change', this.render);
+                    this.render();
```

Comment:
I prefer that views do not call ``render`` in their initialize methods. There are a number of reasons for this:
 - it makes it hard to control when the view is rendered
 - it means that subclasses will render before their own initialization has completed
 - it will render twice if you create a view, configure it, add children etc
 - it doesn't allow for a component to be rendered server-side